docsteady/spec.py

```diff
@@ -70,9 +70,10 @@ def extract_custom_fields(self, data):
 
 class TestCase(Schema):
     key = fields.String(required=True)
+    keyid = fields.Integer()
     name = HtmlPandocField(required=True)
-    owner = fields.Function(deserialize=lambda obj: owner_for_id(obj))
-    owner_id = fields.String(load_from="owner", required=True)
+    owner = fields.Function(deserialize=lambda obj: owner_for_id(obj), default="Unassigned")
+    owner_id = fields.String(load_from="owner")
```

Comment:
If overner_id is required, in case it is not found in the json, we get an error and the document is not generated.
Therefore, we need to let owner_id optional.
We may want to render _Unassigned_ in red in the produced document.